schedule_lessons/dashboard/views.py

```diff
@@ -622,8 +620,15 @@ def my_profile(request):
         "notifications":  [],
         "unread_notifications": len(Notification.objects.filter(
             user=request.user, unread=True)),
-        "blocked_people": []
+        "blocked_people": [],
     }
+    expired_lessons = []
+    if request.user.profile.user_type == 'tutor':
+        context['expired_lessons'] = Lesson.objects.filter(
```

Comment:
just curious is this equivalent to:
```suggestion
        context.expired_lessons = Lesson.objects.filter(
```
and if it is, should we be using the dot notation?